components/reactjs-components/src/grid/tests/__snapshots__/GridRenderer.test.tsx.snap

```diff
@@ -0,0 +1,5 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
do we need this Snapshot thing?

Reply:
This was a copy-paste of the tests from the old grid renderer repo, but these snapshots are incorrect anyway. The snapshots are a good way of testing to make sure nothing has changed in terms of the layout to avoid regressions, but I will fix up these snapshots and tests today.

Reply:
Have fixed the snapshot tests now with some updated jest config.